custom-api/create_data.py

```diff
@@ -0,0 +1,135 @@
+import os
+
+# Databricks SDK
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.service.sql import *
+
+# Initialize Databricks SDK. 
+# By default, this authenticates using the DATABRICKS_HOST and DATABRICKS_TOKEN environment variables
+# initialized by setup.sh.
+w = WorkspaceClient()
+
+warehouse_id = os.environ.get("DATABRICKS_WAREHOUSE_ID")
+
+# This script uses the Statement Execution API to generate sample data.
+
+def execute_statement_sync(sql_statement):
+  response = w.statement_execution.execute_statement(
+    statement = sql_statement,
+    warehouse_id = warehouse_id
+  )
+
+  while response.status.state in (StatementState.PENDING, StatementState.RUNNING):
+    response = w.statement_execution.get_statement(statement_id = response.statement_id)
+
+  if response.status.state != StatementState.SUCCEEDED:
+    print("Failed to execute: '" + sql_statement + "' with state " + str(response.status.state));
+    if response.status.state == StatementState.FAILED:
+        print("Detailed error message: " + response.status.error.message)
+    exit()
+
+  return
+
+# Create a database.
+print("Creating acme_demo database.")
+execute_statement_sync("create database hive_metastore.acme_demo")
```

Comment:
I get this when running setup
```
Failed to execute: 'create database hive_metastore.acme_demo' with state StatementState.FAILED
Detailed error message: [INSUFFICIENT_PERMISSIONS] Insufficient privileges:
User does not have permission CREATE on CATALOG.
```
Am I missing some permissions in dogfood?
I guess I would need create catalog permissions
https://docs.databricks.com/en/data-governance/unity-catalog/manage-privileges/privileges.html

I was able to set this up on BUI dev workspace where I am an admin.

Reply:
Hmm. Only admins can create in the hive_metastore?